dclab/kde/base.py

```diff
@@ -120,6 +125,86 @@ def get_contour(self, xax="area_um", yax="deform", xacc=None, yacc=None,
             xscale=xscale, yscale=yscale
         )
 
+    def get_contour_lines(self, xax="area_um", yax="deform", xacc=None,
+                          yacc=None, kde_type="histogram", kde_kwargs=None,
+                          xscale="linear", yscale="linear", quantiles=None):
+        """Compute contour lines for a given kernel kensity estimate.
+
+        Parameters
+        ----------
+        xax: str
+            Identifier for X axis (e.g. "area_um", "aspect", "deform")
+        yax: str
+            Identifier for Y axis
+        xacc: float
+            Contour accuracy in x direction
+            if set to None, will use :func:`bin_width_doane_div5`
+        yacc: float
+            Contour accuracy in y direction
+            if set to None, will use :func:`bin_width_doane_div5`
+        kde_type: str
+            The KDE method to use
+        kde_kwargs: dict
+            Additional keyword arguments to the KDE method
+        xscale: str
+            If set to "log", take the logarithm of the x-values before
+            computing the KDE. This is useful when data are
+            displayed on a log-scale. Defaults to "linear".
+        yscale: str
+            See `xscale`
+        quantiles: list or array of floats
+            Define the levels of the contours. The values must be between 0
+            and 1. If set to None, will use [0.5, 0.95] as default.
+
+        Returns
+        -------
+        contour_lines: list of lists of arrays
+            For every qualtile level, this list contains a list of
+            corresponding contour lines arrays. Each contour line is a 2D
+            array of shape (N, 2), where N is the number of points in the
+            contour line.
+        qlevel: list of floats
+            The quantile levels for the contour lines. The values
```

Comment:
These are not quantile levels, they are levels corresponding to the values in `quantiles`.